pkg/epinio/pages/c/_cluster/applications/createapp/index.vue

```diff
@@ -111,6 +111,10 @@ export default Vue.extend<Data, any, any, any>({
         // app chart actually belongs in config, so stick it in there
         this.value.configuration = this.value.configuration || {};
         this.set(this.value.configuration, { appchart: appChart });
+        // FIX: this is a hack to get the app chart to show up in the UI
+        if (!!changes.github.url) {
+          this.set(this.value.configuration, { environment: { appDeployment: JSON.stringify(changes.github) } });
```

Comment:
This includes all historic repo, branch and commits which is a lot and we shouldn't need to persist (we have the relevant detail above)

![image](https://user-images.githubusercontent.com/18697775/217800736-2a78b2c3-1220-4811-8909-01ba89f90302.png)

Reply:
This caused all my github build stages to fail with 
```
[stage-dev-sample-app-g-7264d8f886dd3925f31c95d8483ac01ff57ldvzm] fork/exec /cnb/buildpacks/paketo-buildpacks_procfile/5.5.0/bin/detect: argument list too long
```

In the end i manually removed `sourceData` and that seem to work (deployed and app detail gh viewable